pkg/cmd/cli/describe/describer.go

```diff
@@ -689,6 +689,7 @@ func (d *TemplateDescriber) DescribeParameters(params []templateapi.Parameter, o
 	for _, p := range params {
 		formatString(out, indent+"Name", p.Name)
 		formatString(out, indent+"Description", p.Description)
+		formatString(out, indent+"Required", p.Required)
```

Comment:
It seems that `formatString` can't handle booleans. I get:

    $ oc describe template postgresql-persistent
    Name:           postgresql-persistent
    ...
    Parameters:              
        Name:               DATABASE_SERVICE_NAME
        Description:        Database service name
        Required:           %!!(MISSING)s(bool=true)
        Value:              postgresql
    ...